src/blog/pointer-compression.md

```diff
@@ -322,12 +339,25 @@ Also, since we don’t care about sign extending the Smi anymore, this change al
 In terms of the decompression code, it changes a sign-extension operation to a zero-extension, which is just as cheap. However, this simplifies things on the runtime (C++) side. For example, the address space region reservation code (see the [Some implementation details](#some-implementation-details) section).
 
 Here’s the assembly code for comparison:
+
+<!-- markdownlint-disable no-inline-html -->
 :::table-wrapper
-| Decompression | Branchless              | Branchful                    |
-|---------------|-------------------------|------------------------------|
-| Code          | `movsxlq r11,[...]`<br>`testb r11,0x1`<br>`jz done`<br>`addq r11,r13`<br>`done:` | `movl r11,[rax+0x13]`<br>`addq r11,r13`<br><br><br><br> |
-| Summary       | 13 bytes<br>3 or 4 instructions executed<br>1 branch | 7 bytes<br>2 instructions executed<br>no branches |
+<!-- markdownlint-disable no-space-in-code -->
+| Decompression | Branchful                    | Smi-corrupting               |
+|---------------|------------------------------|------------------------------|
+| Code          | ```asm                       | ```asm                       \
+|               |   movsxlq r11,[...]          |   movl r11,[rax+0x13]        \
```

Comment:
There seems to be extra unnecessary spacing in the beginning of each code block line here, unlike in others.

Reply:
Done